### DISCUSSION TITLE
Custom Dataset Valid Thresholding on Training With Skeleton Models

### DISCUSSION THREAD

**Discussion by Spawnfile** [open-mmlab/mmaction2#2715] [General]
Hello everyone,

I'm trying to train skeleton action models with my custom dataset with Pose3D-ntu60-2d and STGCN-ntu60-2d configs. On data extraction code which is provided on github repo of mmaction2 called `ntu_pose_extraction.py` I see a hardcoded resolution which is `(1080, 1920)` on frame shape and video res. My all videos are not in the same shape so I changed this as a dynamic assignment according the input videos shape which is sometimes (1080, 1920) and sometimes (1280, 720) etc. However after doing this the validation script `(valid thresholding`) that comes from `mmengine` ignore all videos of mine.

Additionally before changing dynamic resolution assignment, my about %70 videos of my dataset was ignored also.

here are my questions:
1) Should I hardcode this resolution section even if my videos are in different shapes ?
2) How should I prepare my dataset to get rid of the ignoration from `mmengine`, and what is the working idea of this valid thresholding ?

Cheers
Alper

**Comment by brian123214** [comment]
Were you able to come up with a solution? I also have the same problem.

**Comment by laurahgdrn** [comment]
Did you come up with a solution? I also get the same error.

**Comment by Spawnfile** [comment]
I'm not woking with Mmengine anymore, sorry to announce that.
